pkg/network/encoding/encoding.go

```diff
@@ -55,6 +55,7 @@ func modelConnections(conns *network.Connections) *model.Connections {
 	routeIndex := make(map[string]RouteIdx)
 	httpIndex := FormatHTTPStats(conns.HTTP)
 	httpMatches := make(map[http.Key]struct{}, len(httpIndex))
+	ipc := make(ipCache, len(conns.Conns)/2)
```

Comment:
When do we clear this? Should there be an upper limit on this? LRU?

Reply:
we throw it away once we finish encoding the payload, so there is no reason to worry about capacity or anything like that because it is tied to the lifecycle of the `modelConnections` function call.